src/ducks/balance/components/AccountRow.jsx

```diff
@@ -123,10 +132,13 @@ class AccountRow extends React.PureComponent {
             currencyClassName={styles.AccountRow__figure}
           />
           <Switch
-            defaultChecked
+            disabled={disabled}
+            checked={checked}
             color="primary"
             onClick={this.handleSwitchClick}
             className={styles.AccountRow__switch}
+            id={`${switchIdPrefix}.accounts[${account._id}]`}
```

Comment:
... this way we have `${group.id}.accounts[${account._id}]` here, I find it personnally simpler.

Reply:
I find it a bit unusual to use the HTML id attribute when using event handlers. Is there a particular reason not to use a more conventional :

```
handleChange() {
   const { group, account, onSwitchChange } = this.props
   onSwitchChange(`${group.id}.accounts[${account._id}]`)
}
```

That said, I see no particular reason for not doing this.